bin/common.sh

```diff
@@ -27,9 +29,9 @@ function _rind() {
   if [[ "${has_docker}" == 'true' && "${IN_DOCKER}" != '1' ]]; then
     docker run --rm \
       -v ${ENV_DIR}:${ENV_DIR} \
-      -e CLOUD="$CLOUD" \
-      -e IN_DOCKER="1" \
-      -e CLUSTER="$CLUSTER" \
+      -e CLOUD="${CLOUD}" \
```

Comment:
Now you add accolades that have no effect and are unneeded. I would love to be DRY and aim for least effort coding.

Reply:
@Morriz, I am not aware that I'm repeating myself, I do agree with least effort, but also staying consistent with the style guide.
https://github.com/google/styleguide/blob/gh-pages/shellguide.md#variable-expansion
The styleguide says: In order of precedence: Stay consistent with what you find; quote your variables; prefer "${var}" over "$var".
If you insist, I can change it back to $CLOUD, but then it would be inconsistent with the styleguide.

Reply:
Yeah, I sometimes literally see DRY standing for shrunken bare-minimum code, and the letter R reminds me of Redundant, which is in line with it's philosophy to not repeat yourself typing unnecessary code. Are you still with me?

As discussed, we should add this to the execeptions/additions part of our styleguide.

Reply:
Using good linters syntax errors will be caught. Devs should be schooled in the format of bash vars to differentiate what is var name and what is not.